frontend/src/components/Tag.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react'
+import { type Friend } from './PlanForm'
+import './tag.css'
+
+interface Props {
+  friend: Friend
+  handleRemoveTag: (friend: Friend) => void
+}
+
+export const Tag: React.FC<Props> = ({ friend, handleRemoveTag }) => {
+  const [backgroundColor] = useState<any>(generateRandomLightColor())
+
+  const handleClick = (): void => {
+    handleRemoveTag(friend)
+  }
+  const tagColor = {
+    backgroundColor
+  }
+
+  return (
+    <div>
+      <div className='tag-wrapper' style={tagColor}>
+        <div className='tag-text'>{friend.firstName}</div>
+        <button className='tag-close-button' onClick={handleClick}> x </button>
+      </div>
+    </div>
+  )
+}
+
+// generate light hexademical color string
+function generateRandomLightColor (): string {
```

Comment:
a rare case of a function being named well enough to render the comment redundant! let's rm comment

Reply:
also generally good to pull this logic out into a helper or util file instead